syntax = "proto3";

option csharp_namespace = "Protos";

package photo;
import "google/protobuf/wrappers.proto";
import "google/protobuf/empty.proto";

// The greeting service definition.
service PhotoProto {
  rpc GetPhotoById (google.protobuf.Int32Value) returns (Photo);
  rpc GetAllPhotos (google.protobuf.Empty) returns (Photos);
}

message Photos {
  repeated Photo photos = 1;
}

message Photo {
  int32 albumId = 1;
  int32 id = 2;
  string title = 3;
  string url = 4;
}